scripts/publish_and_release.sh

```diff
@@ -9,22 +9,27 @@ case $RELEASE_TYPE in
    "SNAPSHOT_RELEASE")
    echo "Make SNAPSHOT release"
    ./gradlew publishAllVersions $AUTH
+   git archive --format tar.gz -9 --output api-layer.tar.gz HEAD~1
```

Comment:
Why it has HEAD~1 instead of just HEAD?

Reply:
The Gradle release plugin makes two commits - one before building the code and releasing (HEAD~1) and the second one after it (HEAD).